types/sqlite3/index.d.ts

```diff
@@ -85,4 +86,16 @@ export class Database extends events.EventEmitter {
     on(event: string, listener: (...args: any[]) => void): this;
 }
 
-export function verbose(): void;
+export function verbose(): sqlite3;
+
+// tslint:disable-next-line:strict-export-declare-modifiers
+interface sqlite3 {
```

Comment:
Consider exporting the interface and removing the lint-disable above.

Reply:
This `sqlite3` object is not available in the module itself when using JS; it's just an interface that reresents the structure of the entire module, since I couldn't figure a way to have `verbose()` return `this` (see #8371). As a result,
```
const { sqlite3 } = require('sqlite3')
```
will result in an error.

Should I export it anyway?